azerothcore/mod-learn-spells
# WIP: Add missing spells.

Some spells were ignored by this module's spell filtering algorithm. The algorithm actually seems fine and all of the debugged issue spells come from a invalid `SpellFamily` flag. Seems like blizzard's bug in the `.DBC`. 

## ISSUES ADDRESSED:
- Closes  https://github.com/azerothcore/mod-learn-spells/issues/7

## TESTS PERFORMED:
- Not much yet, WIP. 

## HOW TO TEST THE CHANGES:

- Make a new warrior
- Get level 6
- See if you have the parry spell 
- If you do, yay it fixed parry.

Could also just make the server insta level 80 (or use GM commands) and check on every class if you have anything to learn from a class trainer (should be nothing). 

## KNOWN ISSUES AND TODO LIST:
- [] Add all spells
- [] Fix git
